tests/test_server_basic.py

```diff
@@ -473,6 +473,10 @@ def test_email_oda_api(dispatcher_live_fixture, dispatcher_local_mail_server):
         **default_token_payload
     }
     encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+    
+    if isinstance(encoded_token, bytes):
```

Comment:
Was this case happening ?

Reply:
yes, that's why I added it. It depends on pyjwt version I guess.